test/metabase/models/action_test.clj

```diff
@@ -53,3 +55,16 @@
                        :creator {:common_name "Crowberto Corv"}
                        :parameters [{:id "id" :type :number}]}
                       (hydrate (action/select-action :id action-id) :creator)))))))
+
+(deftest dashcard-deletion-test
+  (mt/test-drivers (mt/normal-drivers-with-feature :actions/custom)
+    (mt/with-actions-test-data-and-actions-enabled
```

Comment:
Is `mt/with-actions-enabled` sufficient? I think `mt/with-actions-test-data-and-actions-enabled` is only needed to test execution of actions.